Pipfile

```diff
@@ -22,7 +22,7 @@ flask = "~=2.2"
 flask-apscheduler = "~=1.12"
 gunicorn = "~=20.1"
 more-itertools = "~=9.1"
-mysql-connector-python = "~=8.0"
+mysql-connector-python = "==8.0.31"
```

Comment:
You shouldn't need to pin this here.  I think something you ran locally updated the version being installed in the `Pipfile.lock` which is the actual log of which versions to install.  Having 8.0.31 installed satisfies the `"~=8.0"` in this `Pipfile` so there's no need for the pin.

Reply:
I have unpinned mysql-connector-python in Pipfile.